vertica_python/vertica/connection.py

```diff
@@ -713,6 +716,26 @@ def read_bytes(self, n):
                 to_read -= received
             return buf
 
+    def read_bytes_using_file(self, n):
```

Comment:
before merging

1. change name to `read_bytes`
2. remove original impl
3. remove line 737

Reply:
done!